logproof/src/fields.rs

```diff
@@ -105,7 +125,7 @@ pub struct SealQ128_1024 {}
  * Fp expects the modulus to be prime, but ours isn't. We need to be good
  * children and not use anything that relies on field primality.
  */
-pub type FqSeal128_8192 = Fp256<MontBackend<SealQ128_8192, 4>>;
+pub type FqSeal128_8192 = Fp192<MontBackend<SealQ128_8192, 3>>;
```

Comment:
SEAL's multiprecision uses the same number of limbs as the number of components in the coefficient modulus, even if that number of limbs higher than required to represent the coefficients. In this case, SEAL will use 4 limbs while the coefficient modulus can fit inside 3 limbs.